packages/cpp/ArmoniK.Api.Worker/header/Worker/TaskHandler.h

```diff
@@ -88,7 +88,7 @@ class TaskHandler {
    * @param data The result data
    * @return A future containing a vector of ResultReply
    */
-  std::future<armonik::api::grpc::v1::agent::ResultReply> send_result(const std::string &key, const std::string &data);
+  std::future<armonik::api::grpc::v1::agent::ResultReply> send_result(const std::string &key, std::string_view data);
```

Comment:
The `key` can also be a `string_view`. That being said, I thought we had a user that is C++14 only?

Reply:
For the worker called by the SDK yeah, but otherwise I think we can use it